components/ipfs/common/ipfs_utils.h

```diff
@@ -13,6 +13,10 @@ namespace ipfs {
 class IpfsUtils {
  public:
   static bool IsIPFSURL(const GURL& url);
+  static bool IsDefaultGatewayURL(const GURL& url);
```

Comment:
all of these should be free functions, no need to put them into a class

Reply:
Thanks for pointing it out, I'll open a small followup for this to minimize the scope here.

Reply:
Follow-up PR at https://github.com/brave/brave-core/pull/6821.